charts/dscp-ipfs-v2/templates/service.yaml

```diff
@@ -0,0 +1,161 @@
+apiVersion: v1
```

Comment:
This brings up three services, I did consider bringing up a perNode service setup as we do in the `dscp-node` chart, however we would not be able to do this using an HPA unless we template based on `autoscaling.maxReplicas` which would create the services and then leave them un-used until an additional ipfs-node is spawned by the autoscaler.  Could be something to add in the future.

Reply:
Yeah so the issue here may be dial up. If they connect to each other over the swam the peerId they connect to will depend on which IPFS they get routed to. This might not work. I think we will eithe rhave to have per node services or not have autoscaling

Reply:
yes

Reply:
This has been fixed, we now have per-node swarm services.